apps/user-profile/README copy.md

```diff
@@ -0,0 +1,61 @@
+# Dev Launchers (https://devlaunchers.org)
+
+## We Build People (and software)
+
+[![](https://dev-launchers.github.io/app/images/dev-launchers__site-screenshot.jpg)](https://devlaunchers.org/)
+
+The site is built with React.js/Next.js and many other wonderful libraries. We're currently using Google Sheets as our backend, but will soon be moving to a dedicated server.
+
+---
+
+## About Dev Launchers
+
+Dev Launchers is a nonprofit tech company working to democratize access to technology and tech related skills. At our core, we build projects and run programs that are designed to prepare the world for a rapidly changing future. We've built an inclusive, software focused incubator giving people from diverse backgrounds the skills and resources necessary to succeed in careers touched by technology, and have an open community focused on advancing technology, one another, and ourselves. This repository holds the beginnings of the online platform we're creating in order to first impart the general skills required for members to begin their own projects, and then support their exploration as they tackle building something they're excited about. It also functions as our organization's website!
+
+---
+
+## Contributing
+
+### Our Issues are organized into different Milestones
+
+_We're using GitHub's milestones to keep better track of our issues. These will change as we progress and move further along with our roadmap. The current milestone's we're tackling are:_
+
+#### [Showing Impact - Let's Bring Life to the Site! (And Get Sponsorships)](https://github.com/dev-launchers/app/milestone/7)
+
+- We want to showcase all the wonderful things Dev Launchers is accomplishing, and the wonderful people making it all happen! This milestone should result in donors and sponsors alike getting a clear idea of what Dev Launchers is all about and why they should support our mission.
+
+#### [Member UX](https://github.com/dev-launchers/app/milestone/9)
+
+- Improve the general experience members have interacting with the site!
+-
+
+#### [General Site Improvements](https://github.com/dev-launchers/app/milestone/8)
+
+- Improve the general presentation and functionality of the site!
+
+---
+
+## Get Started
+
+1. Clone the repo
+
+2. Install: `npm install`
+
+3. Run dev version: `npm run dev`
+
+---
+
+## Release
+
+We are using semantic versioning to tag release. Follow https://github.com/semantic-release/semantic-release#commit-message-format
+to format the commit messages.
+
+Once you are ready to create a new release, create a PR to merge main branch to release branch.
+
+.
+
+---
+
+## LicenseS
```

Comment:
carbon copy of the main readme file, let's make it more suitable to this workspace